src/sentry/tasks/relocation.py

```diff
@@ -1087,6 +1087,12 @@ def postprocessing(uuid: str) -> None:
                 user_id=relocation.owner_id,
                 role="owner",
             )
+
+            terms_accepted.send_robust(
```

Comment:
But the user didn't accept the terms here? We just assume they have, but we should actually record this asap when they do, no?